lib/puppet/provider/netapp_igroup/cmode.rb

```diff
@@ -74,7 +74,13 @@ def members=(value)
     to_add = value - (@original_values[:members] || [])
 
     to_remove.each do |member|
-      igroupremove('initiator-group-name',resource[:name],'initiator',member)
+      force
```

Comment:
https://tickets.puppetlabs.com/browse/MODULES-2592

this is similar to 
https://tickets.puppetlabs.com/browse/MODULES-2579